src/components/plotly-modal/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PlotlyChart from '../plotly-chart';
+import CollapseIcon from '../icons/collapse';
+import BackIcon from '../icons/back';
+import NodeIcon from '../../components/icons/node-icon';
+import { togglePlotModal } from '../../actions';
+import { getClickedNodeMetaData } from '../../selectors/metadata';
+import './plotly-modal.css';
+
+const PlotlyModal = ({ metadata, onToggle, visible }) => {
+  const nodeTypeIcon = metadata?.datasetType || metadata?.node.type;
+
+  const onCollapsePlotClick = () => {
+    // Deselecting a node automatically hides MetaData panel
```

Comment:
this looks like the same comment as mentioned in my review above - please delete